kv/dist_sender_test.go

```diff
@@ -203,8 +203,13 @@ func TestSendRPCOrder(t *testing.T) {
 
 	var testFn rpcSendFn = func(opts rpc.Options, method string,
 		addrs []net.Addr, _ func(addr net.Addr) interface{},
-		_ func() interface{}, _ *rpc.Context) ([]interface{}, error) {
-		return nil, verifyCall(opts, addrs)
+		getReply func() interface{}, _ *rpc.Context) ([]interface{}, error) {
+		err := verifyCall(opts, addrs)
+		if err == nil {
+			replies := append([]interface{}(nil), getReply())
+			return replies, nil
```

Comment:
These two lines can be replaced with `return []interface{}{getReply()}, nil` (here and elsewhere).